packages/atomic/cypress/integration/result-list/result-components/result-fields-list.cypress.ts

```diff
@@ -1,34 +1,56 @@
 import {
-  addTag,
   generateComponentHTML,
   TestFixture,
 } from '../../../fixtures/test-fixture';
 import * as CommonAssertions from '../../common-assertions';
-import {addResultList, buildTemplateWithSections} from '../result-list-actions';
+import {
+  addResultList,
+  buildTemplateWithoutSections,
+} from '../result-list-actions';
 import {
   fieldsListComponent,
   FieldsListSelectors,
 } from './result-fields-list.selector';
 import {resultTextComponent} from './result-text-selectors';
 
-const addInResultListFixture = (numChild: number) => {
+const rem = 16;
+const separatorWidth = rem * 2;
+
+const addInResultListFixture = (
```

Comment:
There was some flakiness in the fields list component with these tests, some ResizeObserver loop errors would appear in the console. I adjusted these tests so that there's exactly enough space for all fields to show (minus one pixel when we want to test whether fields are hidden).